tests/damage_rheology_growth_scaled.prm

```diff
@@ -0,0 +1,150 @@
+# This is a very simple testcase that checks
+# that the implementation of the grain growth
+# laws are correct, and if the work with the 
+# lower mantle grain size scaling. All parameters, including
+# temperature and pressure, are set to constant
+# values and no dynamic recrystallisation is 
+# included (the reciprocal required strain is
+# set to zero). Material flows in from the left
+# and flows out at the right boundary with a constant
+# velocity. The grain size
+# should then only depend on the x coordinate 
+# following the relation
+#
+# d(x) = (d_0 + A * x)^(1/p)
+#
+# with
+# d_0 = 8e-5 m         (initial grain size)
+# p   = 10             (grain size exponent)
+# A   = 2.0294e-46 1/m
+#
+# leading to a grain size of d=8.8957E-5 m at the
+# right boundary of the model. 
```

Comment:
I checked this test, and the output it produces is the same as the one of the damage_rheology_growth test, only scaled by a factor of ten (which is the internal scaling in the test). 

This means that this scaling option is a bit misleading: Internally, the code uses a different grain size to compute grain size evolution than what is given in the output. So I think the test is working as intended, we just have to think about this option again (`Lower mantle grain size scaling`), because I think what it should do is use a scaled grain size internally, but output the correct grain size. On the other hand, that might be difficult to do, as this is based on compositional fields, and we can not just output a different field value from the one the simulation uses.